### PR TITLE
Changes to FlightRecorder and JVM arguments

### PR DESCRIPTION
**Pull request by alanfx**
Remove -XX:+UnlockCommercialFeatures from FlightRecorder options
Update the logging and GC logging options
Update the version for the jdg73 plugin to JDG 7.3.5

### CODE REVIEW

File: core/src/main/java/org/radargun/config/FlightRecorder.java [hunk 1]
```diff
@@ -17,6 +17,9 @@
 
    @Property(doc = "Flight Recorder Options")
    private String flightRecorderOptions;
+   
+   @Property(doc = "Is this an Oracle JVM? Default is false.")
+   private boolean isOracle = false;
```
Comment (rvansa) [inline core/src/main/java/org/radargun/config/FlightRecorder.java]: This is something you can autodetect, see `java.vendor` (regrettably Oracle Corporation for both commercial and OpenJDK) and `java.runtime.name` (that differs).

Comment (alanfx) [reply core/src/main/java/org/radargun/config/FlightRecorder.java]: Thanks, @rvansa! Missed the `java.runtime.name` property

Comment (alanfx) [reply core/src/main/java/org/radargun/config/FlightRecorder.java]: Updated!

**Comment by diegolovison** [comment]
Thanks @alanfx

**Pull request merged**